ansible/roles/install-logging/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+# install-logging tasks
+
+- name: CentOS tasks
+  block:
+    - name: enable powertools repo (CentOS 8)
+      ansible.builtin.command: dnf config-manager --set-enabled powertools
+      when: ansible_distribution_version == "8"
+
+    # There's no CRI-O for RHEL/CentOS 9
```

Comment:
We do not support RHEL 9, so this is better removed, I think

Reply:
Would you like me to remove the "RHEL" text from the comment or all the CentOS 9 tasks?

Reply:
Yes, I think let's remove this. Otherwise, it's confusing.

Reply:
Removed